endurance-page-cache.php

```diff
@@ -93,7 +93,7 @@ public function hooks() {
 			}
 
 			add_action( 'admin_init', array( $this, 'register_cache_settings' ) );
-			add_action( 'save_post', array( $this, 'save_post' ) );
+			add_action( 'transition_post_status', array( $this, 'save_post' ), 10, 3 );
```

Comment:
@earnjam Won't this result in a published page that has been updated NOT being purged?

Reply:
No, `transition_post_status` runs on every call to `wp_insert_post()` whether the post status has changed or not.